src/lxc/tools/lxc_top.c

```diff
@@ -273,6 +279,60 @@ static uint64_t stat_match_get_int(struct lxc_container *c, const char *item,
 	return val;
 }
 
+/*
+examples:
+	blkio.throttle.io_serviced
+	8:0 Read 4259
+	8:0 Write 835
+	8:0 Sync 292
+	8:0 Async 4802
+	8:0 Total 5094
+	Total 5094
+
+	blkio.throttle.io_service_bytes
+	8:0 Read 110309376
+	8:0 Write 39018496
+	8:0 Sync 2818048
+	8:0 Async 146509824
+	8:0 Total 149327872
+	Total 149327872
+*/
+static void stat_get_blk_stats(struct lxc_container *c, const char *item,
+			      struct blkio_stats *stats) {
+	char buf[4096];
```

Comment:
Are you expecting to read a `uint64_t` at max? If so, please use the `LXC_NUMSTRLEN64` macro.